src/mlpack/methods/cf/normalization/combined_normalization.hpp

```diff
@@ -0,0 +1,215 @@
+/**
+ * @file combined_normalization.hpp
+ * @author Wenhao Huang
+ *
+ * CombinedNormalization is a class template for performing a sequence of data
+ * normalization methods which are specified by template parameter.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_CF_NORMALIZATION_COMBINED_NORMALIZATION_HPP
+#define MLPACK_METHODS_CF_NORMALIZATION_COMBINED_NORMALIZATION_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace cf {
+
+/**
+ * This normalization class performs a sequence of normalization methods on
+ * raw ratings.
+ */
+template<typename ...NormalizationTypes>
```

Comment:
Can you attache the ellipsis to the typename, in this case we follow https://msdn.microsoft.com/en-us/library/dn439779.aspx.